triggers/create_incident.ts

```diff
@@ -9,23 +10,23 @@ import createIncidentWorkflow from "../workflows/create_incident.ts";
 const trigger: Trigger<
   typeof createIncidentWorkflow.definition
 > = {
-  type: "shortcut",
+  type: TriggerTypes.Shortcut,
   name: "Create an incident",
   description:
     "Create and send an announcement to one or more channels in your workspace.",
   workflow: "#/workflows/create_incident_workflow",
   inputs: {
     currentUser: {
-      value: "{{data.user_id}}",
+      value: TriggerContextData.Shortcut.user_id,
     },
     currentChannel: {
-      value: "{{data.channel_id}}",
+      value: TriggerContextData.Shortcut.channel_id,
     },
     currentTime: {
       value: "{{event_timestamp}}",
```

Comment:
Oh damn I didn't notice this one - @shapirone think we should add this to the deno-slack-api?

Reply:
@filmaj ☝🏼 Does this need to be waited on before merging?

Reply:
@misscoded nope! I will chat with Neil about that next week separately

Reply:
@filmaj I opened https://github.com/slackapi/deno-slack-api/pull/72 for this, but since these values aren't actually available for every trigger type we held off until we figure out where these values exist and how to communicate/educate this properly. Can discuss more in Slack!